### ISSUE TITLE
Cloud sync iOS

### ISSUE THREAD

**Issue by newuser9290** [LIJI32/SameBoy#628]
Will there ever be any type of cloud sync for iOS? Doesn't have to be iCloud but any type of syncing service to be able to store backups?

**Comment by LIJI32** [comment]
iCloud integration is planned for the next major release. I hoped to get it into 0.16.6, but it appeared to be too complicated to add in a minor release.

**Comment by JD9871** [comment]
Can't wait. Love the way it runs. Appreciate all your hard work the skins make the experience enjoyable

**Comment by jackharro** [comment]
Hi I'd like to add my thoughts

As an end user it looks like there's some API for accessing ‘app documents’ which seems like it would be plug-and-play to replace with iCloud, or any storage provider, depending on the SameBoy codebase. To what extent is this true? How ‘robust’ is the document handling in SameBoy?

In iOS settings there's something called ‘Document Storage’ which lets you pick between your storage providers. I wonder if we can integrate this somehow? I could then use Dropbox and pick up where I left off on my PC, not just iOS devices.

Cheers

**Comment by LIJI32** [comment]
This is under development and mostly complete for the next major release.

**Comment by DerschoeneBernhard** [comment]
Is there a solution for this already? I would love to play the same save state from my iPhone and Desktop PC if possible

**Comment by LIJI32** [comment]
Yes, SameBoy 1.0 introduced iCloud support, letting you move ROMs (and their saves) from local storage to iCloud.
